### ISSUE TITLE
Magic MNPC Vanish!

### ISSUE THREAD

**Issue by 0MrCrazy0** [elBukkit/MagicPlugin#1059]
every time i restart my server i need to type /magic load after i login cos its like my mnpcs dont load. i even had new players join & not get the join kit after i reload plugin everything works. this seems to be a new issue cos it never did this before.

**Comment by Gyztor** [comment]
i know this isnt a perfect fix but it should work temporarily you can use this to have it startup and use that command: https://www.spigotmc.org/resources/ajstartcommands.31033/
i would also post any error logs from the console and the startup logs to see if something is disabling it or breaking it

**Comment by NathanWolf** [comment]
Yes, please check your startup logs for errors .. it would be unusual for the plugin to allow you to use any commands if it failed to load, but it sounds like that's what is happening.

It's also unusual for it to fail to load but then work on a reload... but in any case, please see if there are any errors from the plugin at startup.

**Comment by 0MrCrazy0** [comment]
ajstartcommands is a good workaround i dont see any errors i am using the latest snapshot now as well :) thank so much for your help again.
[log.txt](https://github.com/elBukkit/MagicPlugin/files/8721640/log.txt)
